src/main/java/core/basesyntax/model/FruitModel.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.model;
+
+public enum FruitModel {
+    BANANA("banana"),
+    APPLE(("apple")),
+    LIME("lime"),
+    PEACH("peach"),
+    PINEAPPLE("pineapple");
```

Comment:
if they are the same but in lowercase, not sure if you need to have a code field